### ISSUE TITLE
[Regression] TPS demo on low settings turns white [Intel, GLES3]

### ISSUE THREAD

**Issue by hpvb** [godotengine/godot#26778]
**Godot version:**
9af7fa1

**OS/device including version:**
Linux x64
```
OpenGL vendor string: Intel Open Source Technology Center
OpenGL renderer string: Mesa DRI Intel(R) UHD Graphics 620 (Kabylake GT2) 
OpenGL core profile version string: 4.5 (Core Profile) Mesa 18.3.4
OpenGL core profile shading language version string: 4.50
OpenGL core profile context flags: (none)
OpenGL core profile profile mask: core profile
```

**Issue description:**
Playing the tps-demo with all settings set to 'low' turns the screen completely white after a couple of frames. It seems that the light just intensifies until it fills the screen.

**Steps to reproduce:**
![screenshot from 2019-03-07 15-26-32](https://user-images.githubusercontent.com/2591099/53996595-e4b4ed00-40ed-11e9-819e-9fd54894a412.png)

![screenshot from 2019-03-07 15-26-51](https://user-images.githubusercontent.com/2591099/53996607-f0a0af00-40ed-11e9-9056-fa41e145aba1.png)

![screenshot from 2019-03-07 15-26-53](https://user-images.githubusercontent.com/2591099/53996609-f26a7280-40ed-11e9-94dc-d3a53c12a863.png)

**Minimal reproduction project:**
TPS demo

**Comment by capnm** [comment]
Cool outro-effect idea for losing a game;)

**Comment by qarmin** [comment]
It happens only with GLES 3

**Comment by akien-mga** [comment]
I don't reproduce it on Nvidia GTX 670MX with those settings, might be driver specific.

**Comment by capnm** [comment]
Not reproducible with current master 86d6a72c9745bd15e3217d63b2cc5ba73fe4ad34
on Ubuntu 18.04 + different AMD GPUs. Is also likely a problem with the Intel GPU.

**Comment by Calinou** [comment]
I'll close this issue as nobody has been able to reproduce this recently. Even if it is, it will most likely be fixed in 4.0 with the new renderers.
